validatedeps.js

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env node
```

Comment:
can this go in checkdeps.js - as separate function? seems to make more sense than add another file at the root